packages/chain-adapters/src/ChainAdapterCLI.ts

```diff
@@ -38,6 +38,12 @@ const main = async () => {
   try {
     const chainAdapterManager = new ChainAdapterManager(unchainedUrls)
     const wallet = await getWallet()
+    await wallet.wipe()
+    await wallet.loadDevice({
+      mnemonic: 'all all all all all all all all all all all all',
+      label: 'test',
+      skipChecksum: true
+    })
```

Comment:
Currently mnemonic is loaded from the `.env` file and this now overwrites it. Was the additional `label` or `skipChecksum` needed?